### ISSUE TITLE
[suggestion]: mimeType wildcard support

### ISSUE THREAD

**Issue by HugeLetters** [fabian-hiller/valibot#668]
The way it works currently if you specify expected mimeType as something like `image/*` - it will not match anything really. 

While it would require a more heavy check perhaps if others would also like to see this it would be worthwhile addition. 
It could be created as a separate action like `mimeTypeLoose` to still allow previous behavior.

**Comment by fabian-hiller** [comment]
This has been requested in the past. Honestly, I think most people don't want to accept "any" image, and it's safer to actually specify all the images you want to accept. However, I will think about adding wildcard support in the future.

**Comment by HugeLetters** [comment]
Yeah, I get that if you really need it - it's really not that hard to implement yourself. So my suggestion only makes sense if that's something community would want as whole

**Comment by fabian-hiller** [comment]
I am closing this issue due to inactivity. I welcome feedback if there is still interest in this feature.
